recipes/ryvencore-qt/setup.patch

```diff
@@ -0,0 +1,23 @@
+From 052fb3b0f132cb2bbad9c8ec94f713dc6bb2959e Mon Sep 17 00:00:00 2001
+From: Jan Janssen <jan-janssen@users.noreply.github.com>
+Date: Sat, 15 Jan 2022 22:47:21 -0700
+Subject: [PATCH] Update setup.cfg
+
+---
+ setup.cfg | 3 +--
+ 1 file changed, 1 insertion(+), 2 deletions(-)
+
+diff --git a/setup.cfg b/setup.cfg
+index edc2e37..2e903a8 100644
+--- a/setup.cfg
++++ b/setup.cfg
+@@ -20,8 +20,7 @@ packages = find:
+ include_package_data = True
+ python_requires = >=3.6
+ install_requires = 
+-	ryvencore ==0.3.1
```

Comment:
is this to make `pip check` pass? If that's the case, it might be better to not check with `pip check`?

Reply:
Yes, but then the `pip check` also fails for `ryven` which is based on `ryvencore-qt`.

Reply:
I guess the issue is addressed in https://github.com/conda-forge/pyside2-feedstock/pull/92 - so once that is merged I am happy to update the ryven package to remove the patch.

Reply:
In addition I opened an issue upstream to fix this https://github.com/leon-thomm/ryvencore-qt/pull/17